app/pages/neworg/routes.test.js

```diff
@@ -1,8 +1,8 @@
 import request from 'supertest';
+import FakeAuthProvider from 'buying-catalogue-components/app/common/test/fakeAuthProvider';
```

Comment:
can we do something to tidy this path up a little?

Something like 
import FakeAuthProvider from 'buying-catalogue-components/library';

Reply:
this is the path - would have been easier if you commented on the bcl when I put it in 😄. I will move it out of app and into a folder called library but I was going to have all in their own folder with index file

Reply:
yeah that's fine. In BCL didn't realise the import would look like this 😂

Reply:
fixed it - much better

Reply:
😍